src/intents.cs

```diff
@@ -6497,12 +6959,18 @@ interface INSpeakableStringResolutionResult {
 		INSpeakableStringResolutionResult GetConfirmationRequired (NSObject itemToConfirm, nint reason);
 	}
 
-	[Deprecated (PlatformName.iOS, 13, 0, message: "Use 'INStartCallIntent' instead.")]
+#if XAMCORE_4_0
+	[NoMac]
+#elif MONOMAC
+	[Obsolete ("Unavailable on macOS, will be removed in the future.")]
 	[Deprecated (PlatformName.MacOSX, 10, 15, message: "Use 'INStartCallIntent' instead.")]
+	[Mac (10, 12, 0, PlatformArchitecture.Arch64)]
```

Comment:
it's either a mistake on our side -> `[Obsolete]` since it should never be used
or it's a change on Apple side -> `[Mac]` + `[Deprecated]`
but it should not be both